python/servo/package_commands.py

```diff
@@ -133,6 +137,22 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 checked.update(checking)
                 need_checked.difference_update(checked)
 
+            print("Adding version to Credits.rtf")
+            version_command = [binary_path, '--version']
+            p = subprocess.Popen(version_command, stdout=subprocess.PIPE, universal_newlines=True)
+            version, stderr = p.communicate()
+            if p.returncode != 0:
+                print("Error occurred when getting Servo version: " + stderr)
+                return 1
+            version = "Nightly version: " + version
+
+            template_path = dir_to_resources + 'Credits.rtf.mako'
+            credits_path = dir_to_resources + 'Credits.rtf'
```

Comment:
`os.path.join` is probably a good idea for both of these, even if this is Mac-specific.